posts/2022-02-17-beta-release-post.adoc

```diff
@@ -0,0 +1,200 @@
+---
+layout: post
+title: "Concurrency 3.0 for Jakarta EE 10 in Open Liberty 22.0.0.3-beta"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/ryan-storey
+author_github: https://github.com/ryan-storey
+seo-title: Concurrency 3.0 for Jakarta EE 10 in Open Liberty 22.0.0.3-beta - OpenLiberty.io
+seo-description: Open Liberty 22.0.0.3-beta offers Concurrency 3.0 for Jakarta EE 10, which introduces a variety of new enhancements including resource definition annotations that put applications in control of defining the concurrency constraints and thread context types that the application requires, context-aware completion stages that are managed by the container, and more.
+blog_description: Open Liberty 22.0.0.3-beta offers Concurrency 3.0 for Jakarta EE 10, which introduces a variety of new enhancements including resource definition annotations that put applications in control of defining the concurrency constraints and thread context types that the application requires, context-aware completion stages that are managed by the container, and more.
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= Concurrency 3.0 for Jakarta EE 10 in Open Liberty 22.0.0.3-beta
+Ryan Storey <https://github.com/ryan-storey>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+It's only been a couple of short months since we announced Jakarta EE 9.1 support in Open Liberty 21.0.0.12, and we're already looking forward as we collaborate on and begin to implement Jakarta EE 10 features.  Open Liberty 22.0.0.3-beta offers our first Jakarta EE 10 beta feature, Concurrency 3.0.  Concurrency 3.0 introduces a variety of new enhancements including resource definition annotations that put applications in control of defining the concurrency constraints and thread context types that the application requires, context-aware completion stages that are managed by the container, and more.
```

Comment:
```suggestion
It's only been a few short months since we announced Jakarta EE 9.1 support in Open Liberty 21.0.0.12, and we're already looking forward as we collaborate on and begin to implement Jakarta EE 10 features.  Concurrency 3.0 marks our first Jakarta EE 10 beta feature, available in Open Liberty 22.0.0.3-beta.  Concurrency 3.0 introduces a variety of new enhancements including resource definition annotations that put applications in control of defining the concurrency constraints and thread context types that the application requires, context-aware completion stages that are managed by the container, and more.
```